sx-question.el

```diff
@@ -106,13 +108,14 @@ Each element has the form
 where each element in QUESTION-LIST has the form
 
     (QUESTION_ID . LAST-VIEWED-DATE).")
+(sx-init-variable sx-question--user-read-list)
 
 (defun sx-question--ensure-read-list (site)
   "Ensure `sx-question--user-read-list' has been read from cache.
 If no cache exists for it, initialize one with SITE."
   (unless sx-question--user-read-list
```

Comment:
actually, this entire function is useless now. it's ok for the variable to be nil as long as we know the variable has been initialized (and we do know that now).